samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>During forecasting, the model uses dataset prepared above with NaN values as input with <code>prediction_type</code> as <code>dataframe</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>